experimental/dds/tree2/src/test/feature-libraries/sequence-field/compose.spec.ts

```diff
@@ -1103,4 +1085,21 @@ describe("SequenceField - Compose", () => {
 
 		assert.deepEqual(composed, expected);
 	});
+
+	it("delete (rollback) ○ insert", () => {
+		const insertA = tagChange([Mark.insert(1, brand(0))], tag1);
+		const deleteB = tagRollbackInverse([Mark.delete(1, brand(0))], tag3, tag2);
+		const composed = shallowCompose([deleteB, insertA]);
+
+		// B is the inverse of a new attach. Since that new attach comes after A (temporally),
+		// its tiebreak policy causes the cell to come before A's insert (spatially).
+		// When composing the rollback with A's insert, we the delete should come before the insert,
```

Comment:
Nice comment block btw
```suggestion
		// When composing the rollback with A's insert, the delete should come before the insert,
```